mkosi/__init__.py

```diff
@@ -6108,6 +6110,8 @@ def find_output(args: argparse.Namespace) -> None:
         return
 
     args.output_dir.mkdir(parents=True, exist_ok=True)
+    if args.map_permissions:
+        chown_to_running_user([args.output_dir.parent, args.output_dir])
```

Comment:
shouldn't we chown this only when we created it? and if it already exists, leave it alone?